src/server/async/socket_server.cc

```diff
@@ -1255,6 +1255,7 @@ bool SocketConnection::doSealBlob(json const& root) {
   TRY_READ_REQUEST(ReadSealRequest, root, id);
   RESPONSE_ON_ERROR(bulk_store_->Seal(id));
   RESPONSE_ON_ERROR(bulk_store_->AddDependency(id, getConnId()));
+  server_ptr_->RefreshLists();
```

Comment:
The `SyncObjectEntryList` shouldn't happen every time we seal a new blob, rather, it should be triggered when a "ls ..." action happens.

As creating blobs is though fairly frequent, compared with listing the mounted dir.

Reply:
done